Sources/ComposableArchitecture/ViewStore.swift

```diff
@@ -79,7 +79,7 @@ public final class ViewStore<State, Action>: ObservableObject {
       .sink { [weak self] in
         guard let self = self else { return }
         self.objectWillChange.send()
-        self._state.send($0)
+        self._state.value = $0
```

Comment:
Just out of curiosity, what does this line improve? As I understood from [CurrentValueSubject docs](https://developer.apple.com/documentation/combine/currentvaluesubject) setting `value` is equivalent to calling `send()`.

Reply:
Yeah it's definitely equivalent, and I didn't really need to make this change. The only reason I did it was because `.send` is pretty overloaded in my head to be "viewStore.send" and so I thought we might as well use the simpler syntax for changing the subject's value.

Reply:
Thank you for taking time to explain